### PR TITLE
[Tflite] Add model input, output

### PR DESCRIPTION
**Pull request by zhoonit**
- [Tflite] Add model input, output

```
This patch add model to track input and output

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

File: nntrainer/compiler/tflite_interpreter.cpp [hunk 1]
```diff
@@ -329,6 +361,24 @@ class TfOpIdxMap {
...
+      [&variable_map](const TfOpNode::Variables &variables,
+                      std::vector<int> &v) {
+        for (auto &variable : variables) {
+          auto index = variable_map.getIndex(variable);
+          v.push_back(index);
+        }
+      };
+
+    for (auto &op_node : nodes) {
+      if (op_node.isInput()) {
```
Comment (jijoongmoon) [inline nntrainer/compiler/tflite_interpreter.cpp:375]: isInput is checking if this layer is input layer of network model and isOutput too. so.. It confused little bit. How about change the name such as isInputLayer or so.

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: nntrainer/compiler/tflite_interpreter.cpp [hunk 2]
```diff
@@ -500,9 +526,13 @@ buildSubGraphs(const TfOpNodes &nodes, const TfOpIdxMap &map,
 
   /// @todo extract this to buildSubgraph if there is one or more subgraph
   auto name = fbb.CreateString("main");
+  auto inputs = fbb.CreateVector(map.getInputs());
+  auto outputs = fbb.CreateVector(map.getOutputs());
 
   auto builder = tflite::SubGraphBuilder(fbb);
   builder.add_tensors(tensors);
+  builder.add_inputs(inputs);
+  builder.add_outputs(outputs);
```
Comment (jijoongmoon) [inline nntrainer/compiler/tflite_interpreter.cpp:535]: does Tflite need this inputs and outputs of model?

Comment (zhoonit) [reply nntrainer/compiler/tflite_interpreter.cpp:535]: Yes, they are identified specially.

File: nntrainer/compiler/tflite_interpreter.cpp (hunk 1 above)
Comment (zhoonit) [reply nntrainer/compiler/tflite_interpreter.cpp:375]: I think the reason of confusion comes from we are not explicitly identifying tensor but only operation
But tf identifies tensors and operation.
I'll change to `isInputNode()` :)

Comment (zhoonit) [reply nntrainer/compiler/tflite_interpreter.cpp]: Updated

**Review by lhs8928 — APPROVED** [review]
LGTM

**Pull request merged**